### PR TITLE
Update with internal development

### PR DESCRIPTION
**Pull request by juliaogris**

### CODE REVIEW

File: src/js/lib/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<script src="/src/js/test_data/po_order.model.js"></script>
+<script src="/system/experimental/users/cantosm/po_xml_to_json/sample.json"></script>
+
+
+<body>
+<div id="root"/>
+<script>
+$(() => {
+    const {__, ModelViewer} = sysl_render_model;
+    const model = new PoOrder(sample_data);
```
[the remark this answers is not in the document]
Comment (juliaogris) [reply src/js/lib/index.html]: Apologies, deleted index.hml one directory level up. Want me to reinstate it? Or get rid of js dir altogether?

[the remark this answers is not in the document]
Comment (anzdaddy) [reply src/js/lib/index.html]: Let's just delete this one file.

[the remark this answers is not in the document]
Comment (juliaogris) [reply src/js/lib/index.html]: Ok for now, I'm just a bit wary of half striped and / or disused code. Let's revisit this in the near future

[the remark this answers is not in the document]
Comment (anzdaddy) [reply src/js/lib/index.html]: Agreed. We can make it pri 1 immediately after this and de-bazeling.

[the remark this answers is not in the document]
Comment (anzdaddy) [reply src/js/lib/index.html]: Additionally, I'm not sure that file should exist anyway. It's a demo to validate the model rendering code, and probably shouldn't have been in src/js/lib in the first place.

File: src/importers/import_swagger.py [hunk 2]
```diff
@@ -208,10 +194,10 @@ def info_attrs(info, prefix=''):
             # {foo-bar} to {fooBar}
             w(u'\n{}:', re.sub(r'({[^/]*?})', javaParam, path))
             with w.indent():
-
+
```
[the remark this answers is not in the document]
Comment (juliaogris) [reply src/importers/import_swagger.py]: I have fixed these, I think they resulted from the deleted debug messages. However, I intend to run pylint & .editorconfig over the entire project after de-blazification.

**Comment by juliaogris** [comment]
@anzdaddy, I believe I have completed all requested updates in your code review. As I merge this PR I will squash commit. OK?

File: src/importers/import_swagger.py [hunk 3]
```diff
@@ -193,11 +193,9 @@ def info_attrs(info, prefix=''):
         for (path, api) in sorted(swag['paths'].iteritems()):
             # {foo-bar} to {fooBar}
             w(u'\n{}:', re.sub(r'({[^/]*?})', javaParam, path))
-            with w.indent():
-                
+            with w.indent():
```
Comment (juliaogris) [inline src/importers/import_swagger.py]: Whitespace removed as per codereview

File: src/js/lib/index.html [hunk 4]
```diff
@@ -1,22 +0,0 @@
-<script src="/src/js/lib/vendor/react.js"></script>
```
Comment (juliaogris) [inline src/js/lib/index.html]: File deleted

**Pull request merged**